packages/frontend/src/components/Strategies/Crab/CrabTradeV2.tsx

```diff
@@ -341,14 +410,62 @@ const CrabTradeV2: React.FC<CrabTradeV2Type> = ({ maxCap, depositedAmount }) =>
                 error={!!withdrawError}
               />
             )}
-            <div className={classes.notice}>
-              <div className={classes.infoIcon}>
+            <div className={classes.noticeGray}>
+              <div className={classes.infoIconGray}>
                 <InfoIcon fontSize="medium" />
               </div>
               <Typography variant="caption" color="textSecondary">
                 Crab aims to earn yield in dollar terms. It does this by stacking ETH when price decreases and selling ETH when price increases.
               </Typography>
             </div>
+            { depositFundingWarning && depositOption === 0 ? (
+              <div className={classes.notice}>
+                <div className={classes.infoIcon}>
+                  <Tooltip title={"The strategy sells squeeth to earn yield. Yield is currently lower than usual. You can still deposit, but you may earn less."}>
```

Comment:
Id maybe rephrase this to be "You can still deposit, but you may be more likely to have negative returns or lose money."

Reply:
@alexisgauba

Reply:
good w that!

Reply:
```suggestion
                  <Tooltip title={"The strategy sells squeeth to earn yield. Yield is currently lower than usual. You can still deposit, but you may be more likely to have negative returns."}>
```